.github/workflows/docker-image-push.yml

```diff
@@ -0,0 +1,56 @@
+name: Push Docker image
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build
+    # We want a newer version of qemu: https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1815100
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        dockerfile: [Dockerfile, alpine/Dockerfile]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Docker login
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        run: echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin docker.io
+
+      - name: Setup buildx
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu-user-static
+          docker buildx create --use
+
+      - name: Build Docker image
+        env:
+          PLATFORMS: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
Do we need `linux/arm` and `linux/arm64/v8` in here?

Reply:
`node:12` isn't built for other ARM archs besides `linux/arm64/v8` and `linux/arm/v7` (`node:12-alpine` is also built for `linux/arm/v6`)

Is `linux/arm64/v8` preferred over `linux/arm64`?

Reply:
I have no idea myself.

https://hub.docker.com/_/node/?tab=tags and https://github.com/docker-library/official-images#architectures-other-than-amd64 might be worth investigating.